src/components/Boton.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <button
+    class="checkbox-class"
+    :class="{ 'scaled': isHovered }"
+    @mouseover="isHovered = true"
+    @mouseout="isHovered = false">
+    <slot />
+  </button>
+</template>
+
+<script setup lang = 'ts'>
+
+import {ref} from 'vue';
+
+const isHovered = ref(true);
+
+</script>
+
+<style scoped>
+  .scaled {
+    cursor: pointer;
+    transform: scale(0.9);
+    transition: transform 0.3s ease;
+  }
+
+  /**
+ * Color
+ */
+ .checkbox-class {
+  background-color: #0e4781;
```

Comment:
```suggestion
  background-color: var(--o-color-theme);
```

Hardcodeandolo asi significa que si cambiamos el tema este componente ni se va a inmutar. Tenemos que hacer siempre referencia a las variables CSS de nuestra paleta.